Mathlib/Data/Set/Intervals/OrdConnectedComponent.lean

```diff
@@ -85,13 +85,13 @@ theorem ordConnectedComponent_inter (s t : Set α) (x : α) :
 
 theorem mem_ordConnectedComponent_comm :
     y ∈ ordConnectedComponent s x ↔ x ∈ ordConnectedComponent s y := by
-  rw [mem_ordConnectedComponent, mem_ordConnectedComponent, interval_swap]
+  rw [mem_ordConnectedComponent, mem_ordConnectedComponent, uIcc_swap]
```

Comment:
This `uIcc_swap` seems to be `uIcc_comm` in mathlib3. What is going on here?

Reply:
Whoops sorry, my bad. It should be `uIcc_comm`.